app/assets/javascripts/short-form/templates/d1-preferences-programs.html.slim

```diff
@@ -10,7 +10,7 @@
       .checkbox-group role="group"
         preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE' | translate}}" preference="certOfPreference" labelledby="special_programs_label" dataevent="gtm-preference-cop" translated-description="D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE_DESC"
 
-        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" translated-description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC" ng-if="!listingIs('280 Brighton')"
+        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" translated-description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC" ng-show="!listingIs('280 Brighton')"
```

Comment:
@chrisdolendo note #2: another place where the `ng-if` was F'ing things up (I think we had seen it before w/ these preference directives). Discovered on a test listing that _did_ have DTHP and the ng-if was giving us some `undefined`'s